src/Entity/Aid.php

```diff
@@ -0,0 +1,464 @@
+ * @ORM\Table(name="aids")
+ */
+class Aid
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="uuid", unique=true)
+     * @ORM\GeneratedValue(strategy="CUSTOM")
+     * @ORM\CustomIdGenerator(class=UuidV4Generator::class)
+     */
+    private UuidV4 $id;
+
+    /**
+     * @Gedmo\Timestampable(on="create")
+     * @ORM\Column(type="datetime")
+     */
+    private \DateTimeInterface $createdAt;
+
+    /**
+     * @Gedmo\Timestampable(on="update")
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private ?\DateTimeInterface $updatedAt;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private ?\DateTimeInterface $deletedAt;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $sourceId;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $name;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $fundingType;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $perimeter;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $regionName;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private ?string $goal;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private ?string $beneficiary;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private ?string $aidDetails;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private ?string $eligibility;
+
+    /**
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private ?string $conditions;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $fundingSourceUrl;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private ?\DateTimeInterface $applicationEndDate;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private ?string $applicationUrl;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $state;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     * @Gedmo\Slug(fields={"name"})
+     */
+    private string $slug;
+
+    /**
+     * @ORM\ManyToMany(targetEntity=EnvironmentalAction::class, inversedBy="aids")
+     */
+    private Collection $environmentalActions;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=AidAdvisor::class)
+     */
+    private ?AidAdvisor $aidAdvisor;
+
+    /**
+     * @ORM\ManyToMany(targetEntity=EnvironmentalTopic::class, inversedBy="aids")
+     */
+    private Collection $environmentalTopics;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=Funder::class)
+     */
+    private ?Funder $funder;
+
+    /**
+     * @ORM\ManyToMany(targetEntity=BusinessActivityArea::class)
+     */
+    private Collection $businessActivityAreas;
```

Comment:
Isn't the BusinessActivityArea entity missing the reverse field?

Reply:
For now, there's nowhere on the website asking to find an aid based on the businessActivityArea.
The businessActivityArea is currently working as a tag/filter for the search.